<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_push_back</class>
 <widget class="QMainWindow" name="admin_push_back">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="admin_push_button">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Администратор</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="doctor_push_button">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Врач</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Пользователь</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="users_combo_box"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="user_label">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>316</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
